annotations/src/main/java/com/statful/client/annotations/Timer.java

```diff
@@ -38,4 +38,9 @@
      * @return An array of aggregations to send to Statful
      */
     Aggregation[] aggregations() default {};
+
+    /**
+     * @return An int of the sampleRate to send to Statful
+     */
+    int sampleRate() default 0;
```

Comment:
The enabled/disabled scenario was pretty much just to reinforce that if there's an attribute that allows one to set the sample rate, I think one would expect it to work exactly as set (as in, if set to zero, it should be effectively zero and not some hidden default or some inherited value from another attribute). I don't think there's a use case for enabling/disabling the aspect that makes sense in the scope of the client/aspect.